Microsoft/helium
# Reference App Step 2: Create simple Mocha config

Seems to contain some changes from previous PR's.

To test:
1. Clone code
2. "npm install"
3. "npm run build"
4. "npm test"

Output should show that the example test ran.

To add your test: For now, create your test file next to your source file in src/app, named as "*.test.ts".  The "npm test" command should pick it up.
